src/ethereum_test_tools/consume/base_rpc.py

```diff
@@ -0,0 +1,48 @@
+"""
+Base JSON-RPC class and helper functions for EEST based hive simulators.
+"""
+
+import time
+
+import jwt
```

Comment:
We need to add `PyJWT==2.8.0` (is this the right one?) to `setup.cfg`.

Reply:
> I could not even get it to work lol. How do you set HIVE_SIMULATOR in the vscode config?
I wonder is we should set this automatically, right now its manual:
```
set -x HIVE_SIMULATOR http://127.0.0.1:3000
```
> I think one problem right now is on the src/pytest_plugins/consume/consume.py file with the pytest_configure method, because I don't think this file should try to generate the test cases, and instead this functionality should be moved to a custom pytest_generate_tests in each plugin, consume_direct, consume_via_rlp and consume_via_engine_api.

Nice! This sounds like the move forward - I think currently its this way as we can apply each test case for every consume method.